apps/extension/src/application/trading-copilot/subscriptions-management/components/subscription-form.tsx

```diff
@@ -46,7 +46,7 @@ export const SubscriptionForm = ({ onSubmit }: Properties) => {
     <form onSubmit={form.handleSubmit(addSubscriber)}>
       <label
         htmlFor="subscriptionName"
-        className="block text-sm font-semibold leading-6 text-[#111827]"
+        className="block text-sm font-medium leading-6 text-[#111827]"
```

Comment:
I assume that you didn't want to change too much, but it should have `text-neutralGreen-700 text-label4` to match the designs.